controller_manager/src/ros2_control_node.cpp

```diff
@@ -19,8 +19,8 @@
 #include <thread>
 
 #include "controller_manager/controller_manager.hpp"
-#include "controller_manager/realtime.hpp"
-#include "rclcpp/rclcpp.hpp"
+#include <rclcpp/rclcpp.hpp>
+#include <realtime_tools/thread_priority.hpp>
```

Comment:
We are using `""` for including cpp headers.

```suggestion
#include "rclcpp/rclcpp.hpp"
#include "realtime_tools/thread_priority.hpp"
```

Reply:
If that's the convention, that's fine and I'll make the change.

FYI, `"` is usually used for includes within the package. `<` is usually used for standard, system libraries. It has to do with which directories are searched first.

https://stackoverflow.com/a/21594